src/quo2/foundations/colors.cljs

```diff
@@ -183,6 +183,82 @@
 (def info-50-opa-30 (alpha info-50 0.3))
 (def info-50-opa-40 (alpha info-50 0.4))
 
+;;;;Purple
+(def purple "#8661C1")
```

Comment:
I think this color is already in this file as `purple-50`. have you rebased/ merged the latest from develop in?

Reply:
Thanks Jamie
Will fix it now